### DISCUSSION TITLE
How to trigger workflows when pushed in other workflows?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#26249] [Actions]
<p>Hello.</p>
<p>I’m trying to trigger workflow but always failed</p>
<p>What i want to do is</p>
<p>First, triggering by pull_request. It convert my markdown files to html.</p>
<p>Second, workflow push (or force push) to html files in branch.(my branch is not default branch like main or master. so workflow_run command not works.)</p>
<p>Third. trigger other workflow(do Pytest) after push(or force push) html files</p>
<p>but i always failed at third steps. i tried both workflows. when i did it, the PR lost it’s action result because the commit sha is changed…</p>
<p>Please help me for this. thanks.</p>

**Comment by jsoref** [comment, marked as answer]
<p>If you use a deploy-key and push using that key, you’d be able to respond to the push as a normal workflow event. But be careful not to eat your own tail.</p>

**Comment by ghost** [comment]
<p>Wow. Thank you! I solved my problem and finished my code.</p>
